lib/timers.js

```diff
@@ -409,6 +393,26 @@ function processImmediate() {
   }
 }
 
+function tryOnImmediate(immediate, queue) {
```

Comment:
I changed the one in #4007 to `_tryOnTimeout`, keeping the underspace from a prior incantation in internal timeouts. Mind adding it to this then?